src/lib/pages/backoffice/Actions/CheckOut/CheckOutResults/CheckOutResults.js

```diff
@@ -0,0 +1,52 @@
+import { Loader } from '@components/Loader';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Header, Segment } from 'semantic-ui-react';
+import { ItemList } from './ItemList';
+import { PatronList } from './PatronList';
+
+export default class CheckOutResults extends Component {
+  componentDidMount() {
+    const { clearResults } = this.props;
+    clearResults();
+  }
+
+  /**
+   * NOTE: In case there is a result in itemList or patronList we will be
+   * recirecteded to the relevant details page. If we have more than one results
+   * (exceptional case) we display them so librarian can choose.
+   */
+  render() {
+    const { itemList, isLoading, patronList } = this.props;
+    const hasItems = itemList.length > 1;
```

Comment:
Very minor, about naming... in the case of 1 result, it has items. Maybe something like (shorter?):
```suggestion
    const hasMoreThanOneItem = itemList.length > 1;
```